server/jamovi/core/columnw.cpp

```diff
@@ -170,31 +170,59 @@ void ColumnW::appendLevel(int value, const char *label)
 
     int length = strlen(label)+1;
     size_t allocated;
-
     char *chars = _mm->allocate<char>(length, &allocated);
-
     std::memcpy(chars, label, length);
 
+    length = strlen(importValue)+1;
```

Comment:
you've added a default value to `importValue` of NULL, but you haven't handled the situation that NULL is provided ... i.e. when importValue is NULL it should use the label as the import value.